docs/configmap.md

```diff
@@ -0,0 +1,124 @@
+# Config Map
+
+
+- [Config Map](#config-map)
+  - [About](#about)
+  - [How to use it](#how-to-use-it)
+    - [For OSD](#for-osd)
+    - [For ARO](#for-aro)
+  - [Configurable parameters](#configurable-parameters)
+    - [configManager](#configmanager)
+    - [maintenance](#maintenance)
+    - [scale](#scale)
+    - [upgradeWindow](#upgradewindow)
+    - [nodeDrain](#nodedrain)
+    - [healthCheck](#healthcheck)
+    - [extDependencyAvailabilityChecks](#extdependencyavailabilitychecks)
+
+## About
+The `configmap` which used to tune the `managed-upgrade-operator`. It has various configurable values.
+
+## How to use it
+
+### For OSD
+You can update the `configmap` via [`managed-cluster-config`](https://github.com/openshift/managed-cluster-config/tree/master/deploy/managed-upgrade-operator-config).
+
+### For ARO
+TBD
+
+
+## Configurable knobs
+#### configManager
+
+Please refer to the doc [`configmanager`](../configmanager.md)
```

Comment:
Change `../configmanager.md` to `./configmanager.md` since it's in the `docs` directory itself.